lib/orchestrator.js

```diff
@@ -54,6 +54,30 @@ var Orchestrator = function (url, token, tempLoc, httpLoc, resultLoc) {
 };
 
 /**
+ * Pseudo-constant to indicate a step that finished OK.
+ */
+
+Orchestrator.STATUS_OK = 'ok';
+
+/**
+ * Pseudo-constant to indicate successful completion.
+ */
+
+Orchestrator.STATUS_SUCCESS = 'success';
+
+/**
+ * Pseudo-constant to indicate failure.
+ */
+
+Orchestrator.STATUS_FAILURE = 'failure';
+
+/**
+ * Pseudo-constant to indicate errors.
+ */
+
+Orchestrator.STATUS_ERROR = 'error';
+
```

Comment:
What is it that you don't like here, @astorije?

Starting to substitute some string literals that appear several times in the code with static variables? The documentation comments for those variables? Or the blank lines in between?